### PR TITLE
fix(components): fix assign me and fix color in redelegation

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/pages/faucet/index.tsx [hunk 1]
```diff
@@ -158,19 +159,14 @@ const Faucet = () => {
...
-            alignSelf="flex-start"
+          <AssignMe
             onClick={() => {
               AmpTrack(AmpEvent.USE_ASSIGN_ME);
               setAddress(walletAddress);
             }}
-          >
-            Assign me
-          </Text>
+            isDisable={!walletAddress || address === walletAddress}
```
Comment (poomthiti) [inline src/lib/pages/faucet/index.tsx:167]: remove !walletAddress, the component already checks for this condition

File: src/lib/pages/instantiate/instantiate.tsx [hunk 2]
```diff
@@ -319,18 +320,13 @@ const Instantiate = ({ onComplete }: InstantiatePageProps) => {
...
-                cursor="pointer"
+              <AssignMe
                 onClick={() => {
                   AmpTrack(AmpEvent.USE_ASSIGN_ME);
                   setValue("adminAddress", address);
                 }}
-              >
-                Assign me
-              </Text>
+                isDisable={!address || watchAdminAddress === address}
```
Comment (poomthiti) [inline src/lib/pages/instantiate/instantiate.tsx:328]: ditto

**Review by poomthiti — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -43,6 +43,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Improvements
 
+- [#371](https://github.com/alleslabs/celatone-frontend/pull/371) Refactor assign me component and fix color in redelegation page
```
Comment (evilpeach) [inline CHANGELOG.md:46]: Please rebase from develop branch in order to fix the confilct.

**Review by evilpeach — APPROVED** [review]
lgtm

**Review by poomthiti — APPROVED** [review]

**Pull request merged**